packages/evolution-common/src/services/baseObjects/WorkPlace.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+
+import { Optional } from '../../types/Optional.type';
+import { IValidatable } from './IValidatable';
+import * as PlAttr from './attributeTypes/PlaceAttributes';
+import { ParamsValidatorUtils } from '../../utils/ParamsValidatorUtils';
+import { Place, PlaceAttributes, placeAttributes } from './Place';
+import { Result, createErrors, createOk } from '../../types/Result.type';
+
+/*
+ * A work place is a place associated with a person
+ * where they go to work on a regular basis (usual work place)
+ */
+
+export const workPlaceAttributes = [
+    ...placeAttributes,
+    'parkingType',
```

Comment:
How is this different from a school place? can't they just all be places with a type instead?

Reply:
For now, this is quite similar, but later on, we will add more specific attributes, like number of student/number of employee, etc.